### DISCUSSION TITLE
How to use Multi-Speaker Tacotron2?

### DISCUSSION THREAD

**Discussion by hemath1001** [coqui-ai/TTS#1389] [General Q&A]
I find some models in [Released-Models](https://github.com/coqui-ai/TTS/wiki/Experimental-Released-Models) are not included in .model.json, such as [Multi-Speaker-Tacotron2](https://drive.google.com/drive/folders/1wGpFgHBHahgCgUrmjxpjK80Reb1Uh5V_?usp=sharing) and [Multi-Speaker-Tacotron2 DDC](https://drive.google.com/drive/folders/1HggGhpYszc4H2fmbxfMM4wCAMNaUKdQH?usp=sharing).
I have downloaded and run these 2 models, but it seems that they can only run with the specific old coqui version and with their own code in colab? Why they are forgotten by current coqui version...
Besides, I'm wondering whether every multi-speaker TTS model have the specific trainer encoder model. If I make another encoder model work with a multi-speaker TTS model, the effect will be discounted, right?

**Comment by erogol** [comment]
https://tts.readthedocs.io/en/dev/inference.html

**Reply by hemath1001** [reply]
Thanks @erogol ! I had checked the doc. My question is: It seems [Multi-Speaker-Tacotron2](https://drive.google.com/drive/folders/1wGpFgHBHahgCgUrmjxpjK80Reb1Uh5V_?usp=sharing) and [Multi-Speaker-Tacotron2 DDC](https://drive.google.com/drive/folders/1HggGhpYszc4H2fmbxfMM4wCAMNaUKdQH?usp=sharing) can not run with current coqui. They can only run with the specific old coqui version and with their own code in colab(the code for inference is different).
